src/core/variables.tf

```diff
@@ -423,3 +423,30 @@ variable "selfcare_plan_sku_capacity" {
   type        = number
   default     = 1
 }
+
+
+# legal backup storage
+variable "legalbackup_account_replication_type" {
+  type        = string
+  description = "Legal backup replication type"
+  default     = "LRS"
```

Comment:
if this storage will contains legal backup file I suggest to use GRS (Global Redundancy Storage) to replicate data in a secondary region

Reply:
change it here https://github.com/pagopa/io-infra/pull/84/commits/77727f1ec8e24c2c4f8b248a989b9614600345a7